Lib/ufo2ft/_compilers/interpolatableOTFCompiler.py

```diff
@@ -0,0 +1,59 @@
+import dataclasses
+from dataclasses import dataclass
+from typing import Optional, Type
+
+from fontTools import varLib
+
+from ufo2ft.constants import SPARSE_OTF_MASTER_TABLES, CFFOptimization
+from ufo2ft.outlineCompiler import OutlineOTFCompiler
+from ufo2ft.preProcessor import OTFPreProcessor
+
+from .baseCompiler import BaseInterpolatableCompiler
+from .otfCompiler import OTFCompiler
+
+
+@dataclass
+class InterpolatableOTFCompiler(OTFCompiler, BaseInterpolatableCompiler):
```

Comment:
this double inheritance, what does it actually accomplish? It's not immediately clear what the effects of this are. Maybe a comment would help